Source/Core/DolphinWX/MainNoGUI.cpp

```diff
@@ -366,6 +368,21 @@ int main(int argc, char* argv[])
 
   platform->Init();
 
+  // Shut down cleanly on SIGINT and SIGTERM
+  struct sigaction sa;
+  sa.sa_handler = [](int unused) {
```

Comment:
Don't make that a lambda. It happens to work here but in practice I don't know if the standard specifies what you're doing. That lambda object should get "deleted" when "sa" leaves the scope.